components/frontend/src/utils.js

```diff
@@ -271,7 +271,7 @@ export function nice_number(number) {
         }
         rounded_numbers = rounded_numbers.map((value) => { return value * 10 });
     }
-    while (true);
+    while (true);  // eslint-disable-line no-constant-condition
```

Comment:
What would be the eslint acceptable implementation for this? Perhaps worth doing in a separate issue?

Reply:
I'm not sure there is one?